beetsplug/convert.py

```diff
@@ -132,6 +133,7 @@ def __init__(self):
             u'paths': {},
             u'never_convert_lossy_files': False,
             u'copy_album_art': False,
+            u'copy_album_art_maxwidth': 0,
```

Comment:
How about just `art_maxwidth` or `album_art_maxwidth` if you prefer? "Copy" seems a little redundant.